### ISSUE TITLE
[Suggestion] Integration with Overwolf Overlay (App)

### ISSUE THREAD

**Issue by orloko** [SilentSys/SLAM#37]
Will be amazing if we can select/organize musics via Overlay.

**Comment by TypicalHog** [comment]
I agree, but tags are much better when you learn them all + there is no room on the screen for all of my 743 tracks, I would just hope Andy handles tag collisions in some way, shape or form 'cause it's the most annoying thing for me currently.
